sdks/java/core/src/main/java/org/apache/beam/sdk/coders/CoderRegistry.java

```diff
@@ -235,7 +235,9 @@ public void registerCoderForType(TypeDescriptor<?> type, Coder<?> coder) {
    *
    * @throws CannotProvideCoderException if a {@link Coder} cannot be provided
    */
+  // CHECKSTYLE.OFF: MissingDeprecated
   @Deprecated
+  // CHECKSTYLE.ON: MissingDeprecated
```

Comment:
`getCoderFromDescriptor` doesn't exist, do you mean the private `getCoderFromTypeDescriptor`?.

Also, this method has no replacement and is really meant to be copied into `ParDo` having `ParDo` own all that logic.